README.md

```diff
@@ -1,2 +1,37 @@
-# slack
-👨‍💻 Sign up page for our slack
+
+<br>
+<p align="center">
+  <img alt="HackCU IV" src="https://github.com/HackCU/splash-page/blob/master/img/hackcu_black.png" width="200"/>
+</p>
+<br>
+
+Slack Sign Up Page
+
+# Run project
+
+Needs: Ruby
+
+## Local 
+
+1. `git clone https://github.com/HackCU/slack.git && cd slack`
+2. `gem install jekyll`
+3. `jekyll serve --watch`
+
+## Deploy
+
+1. Push changes to master
+2. There's no `2`
+
+Deployment is done automatically by [GitHub pages](https://pages.github.com/). You don't need to do anything else than pushing to master.
+
+# Development
+
+## To add new logo images for sponsors and partners
+
+1. Remove background
+1. Resize the image to `600 x 400 px` centering
+3. Archive the original `.png` file in the corresponding `originals/` folder
```

Comment:
No logos on this page most probably, you can delete this lines.

Reply:
👍